### ISSUE TITLE
How can we run the SIG without requiring everyone meet synchronously?

### ISSUE THREAD

**Issue by JamieDanielson** [open-telemetry/sig-contributor-experience#14]
How do we run SIGs without all decisions being made in one place (a single SIG meeting), and ensure knowledge is passed around approprirately?

**Comment by codefromthecrypt** [comment]
Good idea. One input to this would be to ask each or at least multiple SIGs if they feel status quo is desired. I've heard anecdotally that some SIGs prefer decisions made in one meeting, but don't know for sure which or how many prefer this.

**Comment by mx-psi** [comment]
Is this specifically about the Contributor Experience SIG or generally about all SIGs?

**Comment by theletterf** [comment]
I suggest moving this one to Discussions
